src/scripts/main.js

```diff
@@ -2,4 +2,44 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+
+  if (e.target !== wall) {
+    return;
+  }
+
+  const clickX = e.clientX;
+  const clickY = e.clientY;
+
+  const leftBorder = wall.offsetLeft + wall.clientLeft;
+  const topBorder = wall.offsetTop + wall.clientTop;
```

Comment:
The calculation of `leftBorder` and `topBorder` includes `wall.clientLeft` and `wall.clientTop`, which are usually zero unless the element has a border. Ensure that these values are necessary for your layout. If the wall has no border, you might want to remove these terms to avoid incorrect positioning of the spider.